pkg/testutils/mockmaps/lbmap.go

```diff
@@ -16,29 +16,37 @@ import (
 
 type LBMockMap struct {
 	lock.Mutex
-	BackendByID            map[lb.BackendID]*lb.Backend
-	ServiceByID            map[uint16]*lb.SVC
-	AffinityMatch          datapathTypes.BackendIDByServiceIDSet
-	SourceRanges           datapathTypes.SourceRangeSetByServiceID
-	DummyMaglevTable       map[uint16]int // svcID => backends count
+	BackendByID   map[lb.BackendID]*lb.Backend
+	ServiceByID   map[uint16]*lb.SVC
+	AffinityMatch datapathTypes.BackendIDByServiceIDSet
+	SourceRanges  datapathTypes.SourceRangeSetByServiceID
+	// metrics for upsertMaglevLookupTable mock
+	UpsertMaglevLookupTableCallCnt map[uint16]int
+	DummyMaglevTable               map[uint16]int // svcID => backends count
+
+	// metrics for UpsertService mock
+	UpsertServiceCallCnt   map[uint16]int
```

Comment:
So basically when we noticed there's "something" wrong... it just looked as if the functions that were supposed to be called (upsertService or upsertMaglevLookupTable) were completely skipped.

I think I left it, since it could be useful, but sure, I'll remove it.